webpack/webpack.config.js

```diff
@@ -0,0 +1,92 @@
+var ExtractTextPlugin = require('extract-text-webpack-plugin');
+module.exports = {
+  entry: './app/upload.js',
+  output: {
+    filename: 'upload.js',
+    path: '../src/collective/upload/browser/static',
```

Comment:
that's weird.. here this problem don't happen.

also, you can call just `webpack` instead of `../bin/webpack` since `webpack_env` script put all content of `bin` folder at `$PATH`